ph5/core/ph5api.py

```diff
@@ -910,37 +910,38 @@ def cut (self, das, start_fepoch, stop_fepoch, chan=1, sample_rate=None, apply_t
             window_samples = d['sample_count_i']
             #   Window stop epoch
             window_stop_fepoch = window_start_fepoch + (window_samples / sr)
+
             #   Requested start before start of window, we must need to start cutting at start of window
             if start_fepoch < window_start_fepoch :
                 cut_start_fepoch = window_start_fepoch
                 cut_start_sample = 0
             else :
             #   Cut start is somewhere in window
                 cut_start_fepoch = start_fepoch
-                cut_start_sample = int ((start_fepoch - window_start_fepoch) * sr)
+                cut_start_sample = round((((start_fepoch*1000000 - window_start_fepoch*1000000))/1000000 * sr))
```

Comment:
Why do you multiply by 1000000 here?

Reply:
That doesn't really need to be there.  That was probably left there when I was testing where rounding errors were occurring. We should probably change that back before creating the release.

Reply:
Okay thanks. Can you please change it back?

Reply:
Sure I have a meeting right now, but should have it fixed within the hour

Reply:
Thanks Derick.